File: CS/WindowsApplication41/RatingControlInfoArgs.cs
using System;
using System.Drawing;
using System.Drawing.Drawing2D;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;
using DevExpress.LookAndFeel;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.ViewInfo;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.Drawing;
using DevExpress.XtraPrinting;
using DevExpress.XtraEditors.Registrator;
using DevExpress.Utils.Drawing;
using DevExpress.Utils;

public class RatingControlObjectInfoArgs : ObjectInfoArgs {
    public RatingControlObjectInfoArgs() { }
    public Color StarsRectangleBackgroundColor;
    public Color TitlesRectangleBackgroundColor;
    public Color NormalStarColor;
    public Color SelectedStarColor;
    public Color HotTrackedStarColor;
    public string Title;
    public Color BackgroundColor;
    public Color BarColor;
    public float Percent = 0;
    public int Value;
    public int HotTrackValue = 0;
    
}

File: VB/WindowsApplication41/RatingControlInfoArgs.vb
Imports System.Drawing
Imports System.Drawing.Drawing2D
Imports System.ComponentModel
Imports DevExpress.XtraEditors.Drawing
Imports DevExpress.Utils.Drawing

Public Class RatingControlObjectInfoArgs
    Inherits ObjectInfoArgs

    Public Sub New()
    End Sub

    Public StarsRectangleBackgroundColor As Color

    Public TitlesRectangleBackgroundColor As Color

    Public NormalStarColor As Color

    Public SelectedStarColor As Color

    Public HotTrackedStarColor As Color

    Public Title As String

    Public BackgroundColor As Color

    Public BarColor As Color

    Public Percent As Single = 0

    Public Value As Integer

    Public HotTrackValue As Integer = 0
End Class
